Build on Ubuntu 18.04 (bionic) fails

I am not able to get a build of the examples files on U18.04 bionic. I am building outside of the source directory, which is in /usr/share/doc/dart/examples (owned by root).

using cmake v 3.10.2

Here are the steps I've taken:
```
% sudo apt-get install libdart6-all-dev
% cd ~/code/dart
% md build
% cd build
% cmake /usr/share/doc/dart/examples
```
I get the following error:

```
jeff@dragon-vm:~/code/dartex/build$ cmake /usr/share/doc/dart/examples
-- The C compiler identification is GNU 7.3.0
-- The CXX compiler identification is GNU 7.3.0
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Found DART: /usr/include (Required is at least version "6.5.0") found components:  utils-urdf gui dart 
-- Found DART: /usr/include (Required is at least version "6.5.0") found components:  utils-urdf gui dart 
-- Found DART: /usr/include (Required is at least version "6.5.0") found components:  utils-urdf gui dart 
-- Found DART: /usr/include (Required is at least version "6.5.0") found components:  utils-urdf gui dart 
-- Found DART: /usr/include (Required is at least version "6.5.0") found components:  utils-urdf gui dart 
-- Found DART: /usr/include (Required is at least version "6.5.0") found components:  utils-urdf gui dart 
-- Found DART: /usr/include (Required is at least version "6.5.0") found components:  utils-urdf gui dart 
-- Found DART: /usr/include (Required is at least version "6.5.0") found components:  utils-urdf gui dart 
-- Found DART: /usr/include (Required is at least version "6.5.0") found components:  utils-urdf gui dart 
-- Found DART: /usr/include (Required is at least version "6.2.0") found components:  utils-urdf gui dart 
-- Found DART: /usr/include (Required is at least version "6.5.0") found components:  utils-urdf gui dart 
-- Found DART: /usr/include (Required is at least version "6.5.0") found components:  utils-urdf gui dart 
-- Found DART: /usr/include (Required is at least version "6.5.0") found components:  utils-urdf gui dart 
CMake Error at simpleFrames/CMakeLists.txt:17 (add_executable):
  add_executable called with incorrect number of arguments

CMake Error at simpleFrames/CMakeLists.txt:19 (target_link_libraries):
  Cannot specify link libraries for target "simpleFrames" which is not built
  by this project.

-- Configuring incomplete, errors occurred!
See also "/home/jeff/code/dartex/build/CMakeFiles/CMakeOutput.log".
```
Any suggestions as to what I might be doing wrong are appreciated.

yes, can't seem to find the srcs.

here's the output.
the examples are installed in /`usr/share/doc/dart/examples`, and all owned by root

```
jeff@dragon-vm:~/code/dartex/build$ cmake /usr/share/doc/dart/examples
-- Found DART: /usr/include (Required is at least version "6.5.0") found components:  utils-urdf gui dart 
-- Found DART: /usr/include (Required is at least version "6.5.0") found components:  utils-urdf gui dart 
-- Found DART: /usr/include (Required is at least version "6.5.0") found components:  utils-urdf gui dart 
-- Found DART: /usr/include (Required is at least version "6.5.0") found components:  utils-urdf gui dart 
-- Found DART: /usr/include (Required is at least version "6.5.0") found components:  utils-urdf gui dart 
-- Found DART: /usr/include (Required is at least version "6.5.0") found components:  utils-urdf gui dart 
-- Found DART: /usr/include (Required is at least version "6.5.0") found components:  utils-urdf gui dart 
-- Found DART: /usr/include (Required is at least version "6.5.0") found components:  utils-urdf gui dart 
-- Found DART: /usr/include (Required is at least version "6.5.0") found components:  utils-urdf gui dart 
-- Found DART: /usr/include (Required is at least version "6.2.0") found components:  utils-urdf gui dart 
-- Found DART: /usr/include (Required is at least version "6.5.0") found components:  utils-urdf gui dart 
-- Found DART: /usr/include (Required is at least version "6.5.0") found components:  utils-urdf gui dart 
-- Found DART: /usr/include (Required is at least version "6.5.0") found components:  utils-urdf gui dart 
CMake Error at simpleFrames/CMakeLists.txt:17 (add_executable):
  add_executable called with incorrect number of arguments

-- [DEBUG] PROJECT_NAME: simpleFrames
-- [DEBUG] srcs: 
CMake Error at simpleFrames/CMakeLists.txt:21 (target_link_libraries):
  Cannot specify link libraries for target "simpleFrames" which is not built
  by this project.

-- Configuring incomplete, errors occurred!
See also "/home/jeff/code/dartex/build/CMakeFiles/CMakeOutput.log".

```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,5 +1,11 @@
 ## DART 6
 
+### [DART 6.6.1 (2018-08-04)](https://github.com/dartsim/dart/milestone/46?closed=1)
+
+* Examples
+
+  * Fix CMakeLists.txt of humanJointLimits: [#1094](https://github.com/dartsim/dart/pull/1094)
+
 ### [DART 6.6.0 (2018-08-02)](https://github.com/dartsim/dart/milestone/44?closed=1)
 
 * Collision detection

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -56,7 +56,7 @@ endif()
 # If you change the version, please update the <version> tag in package.xml.
 set(DART_MAJOR_VERSION "6")
 set(DART_MINOR_VERSION "6")
-set(DART_PATCH_VERSION "0")
+set(DART_PATCH_VERSION "1")
 set(DART_VERSION "${DART_MAJOR_VERSION}.${DART_MINOR_VERSION}.${DART_PATCH_VERSION}")
 set(DART_PKG_DESC "Dynamic Animation and Robotics Toolkit.")
 set(DART_PKG_EXTERNAL_DEPS "eigen, ccd, fcl, assimp, boost")

--- a/examples/humanJointLimits/CMakeLists.txt
+++ b/examples/humanJointLimits/CMakeLists.txt
@@ -9,7 +9,7 @@ project(humanJointLimits)
 
 find_package(DART 6.4 REQUIRED
   COMPONENTS
-    dart-utils-urdf dart-gui dart-collision-ode dart-collision-bullet
+    utils-urdf gui collision-ode collision-bullet
   CONFIG
 )
 

--- a/examples/rigidCubes/CMakeLists.txt
+++ b/examples/rigidCubes/CMakeLists.txt
@@ -7,7 +7,7 @@ endif()
 
 project(rigidCubes)
 
-find_package(DART 6.2.0 REQUIRED COMPONENTS utils-urdf gui CONFIG)
+find_package(DART 6.6.0 REQUIRED COMPONENTS utils-urdf gui CONFIG)
 
 add_compile_options(-std=c++11)
 

--- a/package.xml
+++ b/package.xml
@@ -4,7 +4,7 @@
        a Catkin workspace. Catkin is not required to build DART. For more
        information, see: http://ros.org/reps/rep-0136.html -->
   <name>dartsim</name>
-  <version>6.6.0</version>
+  <version>6.6.1</version>
   <description>
     DART (Dynamic Animation and Robotics Toolkit) is a collaborative,
     cross-platform, open source library created by the Georgia Tech Graphics

